.github/workflows/ci.yml

```diff
@@ -0,0 +1,83 @@
+name: CI
+
+on:
+  push:
+    branches: [ main ]
```

Comment:
This should be `master` right?

Reply:
Indeed.

Reply:
Resolved in [d30a47a](https://github.com/ParthJadhav/rust_search/pull/30/commits/d30a47a1c6db98b981c4a49e17604c3acc259fa5).